t/integration/centos7-to-almalinux8/complete.t

```diff
@@ -7,13 +7,11 @@
 
 use lib '/usr/local/cpanel/';
 
-use Cpanel::OS              ();
-use Cpanel::SafeRun::Simple ();
+use Cpanel::OS ();
 
-use Test::More tests => 2;
```

Comment:
I removed the number of expected tests to be more in line with how this is handled throughout the rest of the elevate project.  Basically, this project relies on using `done_testing` rather than enumerating the tests pretty much all of its unit tests so it made more sense to me to keep that consistency.